_topic_maps/_topic_map.yml

```diff
@@ -927,37 +927,47 @@ Topics:
 - Name: Compliance Operator
   Dir: compliance_operator
   Topics:
+  - Name: Compliance Operator overview
+    File: co-overview
   - Name: Compliance Operator release notes
     File: compliance-operator-release-notes
-  - Name: Supported compliance profiles
-    File: compliance-operator-supported-profiles
-  - Name: Installing the Compliance Operator
-    File: compliance-operator-installation
-  - Name: Updating the Compliance Operator
-    File: compliance-operator-updating
-  - Name: Compliance Operator scans
-    File: compliance-scans
-  - Name: Understanding the Compliance Operator
-    File: compliance-operator-understanding
-  - Name: Managing the Compliance Operator
-    File: compliance-operator-manage
-  - Name: Tailoring the Compliance Operator
-    File: compliance-operator-tailor
-  - Name: Retrieving Compliance Operator raw results
-    File: compliance-operator-raw-results
-  - Name: Managing Compliance Operator remediation
-    File: compliance-operator-remediation
-  - Name: Performing advanced Compliance Operator tasks
-    File: compliance-operator-advanced
-  - Name: Troubleshooting the Compliance Operator
-    File: compliance-operator-troubleshooting
-  - Name: Uninstalling the Compliance Operator
-    File: compliance-operator-uninstallation
-  - Name: Using the oc-compliance plugin
-    File: oc-compliance-plug-in-using
-  - Name: Understanding the Custom Resource Definitions
-    File: compliance-operator-crd
-
+  - Name: Compliance Operator concepts
+    Dir: co-concepts
+    Topics:
+    - Name: Understanding the Compliance Operator
+      File: compliance-operator-understanding
+    - Name: Understanding the Custom Resource Definitions
+      File: compliance-operator-crd
+    - Name: Managing the Compliance Operator
+      File: compliance-operator-manage
+  - Name: Compliance Operator management
+    Dir: co-management
+    Topics:
+    - Name: Installing the Compliance Operator
+      File: compliance-operator-installation
+    - Name: Updating the Compliance Operator
+      File: compliance-operator-updating
+    - Name: Troubleshooting the Compliance Operator
+      File: compliance-operator-troubleshooting
```

Comment:
Is this going to evolve into a troubleshooting guide for installation, upgrades, and uninstallation?

I'm wondering because the current troubleshooting [guide](https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-troubleshooting.html) is full of troubleshooting tips about non-operator lifecycle tasks (install, upgrade, uninstall).

Reply:
We may need to consider two troubleshooting pages, one for Operator tasks and one for scan-related tasks. For now, I'll move this to the CO scan management section.